src/apply/apply.go

```diff
@@ -361,22 +348,6 @@ func insertCustomApp(jsPath string, flags Flag) {
 	})
 }
 
-func insertHomeConfig(jsPath string, flags Flag) {
-	if !flags.HomeConfig {
-		return
-	}
-
-	utils.ModifyFile(jsPath, func(content string) string {
-		utils.ReplaceOnce(
-			&content,
-			`([\w$_\.]+\.sections\.items)(\.map)`,
```

Comment:
Is this the only regex for home config?

Reply:
yes

Reply:
So, what do you mean when you say "it requires multiple regexes to patch home~v2"?

Reply:
it would require multiple regexes to work on the versions we support

Reply:
I'd be more comfortable in marking this as help needed to see if anyone can help than simply removing the feature I think

Reply:
well, okay... just saying that no one uses it i think because only one person in last few months asked about it 🤷‍♀️